packages/web/src/brandkit/IconsPage.tsx

```diff
@@ -1,81 +1,64 @@
+import fetch from 'cross-fetch'
+import { NextPage } from 'next'
 import * as React from 'react'
 import { StyleSheet, View } from 'react-native'
 import CCLicense from 'src/brandkit/common/CCLicense'
-import { brandStyles, GAP } from 'src/brandkit/common/constants'
-import Fetch from 'src/brandkit/common/Fetch'
+import { GAP } from 'src/brandkit/common/constants'
 import Page, { ICONS_PATH } from 'src/brandkit/common/Page'
 import PageHeadline from 'src/brandkit/common/PageHeadline'
-import IconShowcase from 'src/brandkit/common/Showcase'
-import { AssetTypes } from 'src/brandkit/tracking'
-import { I18nProps, NameSpaces, withNamespaces } from 'src/i18n'
+import { NameSpaces, useTranslation } from 'src/i18n'
 import { hashNav } from 'src/shared/menu-items'
+import { Explorer } from './Explorer'
 
-export default React.memo(
-  withNamespaces(NameSpaces.brand)(function IconsPage({ t }: I18nProps) {
-    return (
-      <Page
-        title={t('icons.title')}
-        metaDescription={t('icons.headline')}
-        path={ICONS_PATH}
-        sections={[{ id: hashNav.brandIcons.overview, children: <Overview /> }]}
-      />
-    )
-  })
-)
+export interface Props {
+  icons: IconData[]
+}
 
-interface IconData {
+const IconPage: NextPage<Props> = React.memo(function IconsPage({ icons }: Props) {
+  const { t } = useTranslation(NameSpaces.brand)
+  return (
+    <Page
+      title={t('icons.title')}
+      metaDescription={t('icons.headline')}
+      path={ICONS_PATH}
+      sections={[{ id: hashNav.brandIcons.overview, children: <Overview icons={icons} /> }]}
+    />
+  )
+})
+
+IconPage.getInitialProps = async ({ req }) => {
+  let icons = []
+  if (req) {
```

Comment:
what is req?

Reply:
excellent question! 

`req` is request like in express, basically its how to tell we running this serverside or client side.  which is important since if client side we will need to send a request to our endpoint, and if serverside we want to call the function directly. But we want to NOT import that function on client to save our bundle size (and it wont work) so we do an awaited import

Reply:
if req exists its being run on a server, if not its being run in browser